src/ruby/MassMatrix3.i

```diff
@@ -103,6 +103,17 @@ namespace gz
       public: bool SetFromCylinderZ(const T _length,
                                     const T _radius,
                                     const Quaternion<T> &_rot);
+      public: bool SetFromConeZ(const Material &_mat,
+                                    const T _length,
+                                    const T _radius,
+                                    const Quaternion<T> &_rot = Quaternion<T>::Identity);
+      public: bool SetFromConeZ(const T _mass,
+                                    const T _length,
+                                    const T _radius,
+                            const Quaternion<T> &_rot = Quaternion<T>::Identity);
+      public: bool SetFromConeZ(const T _length,
+                                    const T _radius,
+                                    const Quaternion<T> &_rot);
```

Comment:
indentation

Reply:
Fixed it as well as indentation for SetFromCylinderZ that it was copied off of:
https://github.com/gazebosim/gz-math/blob/118af0968fcb131338f9ebb9bba881ce43dbaa96/src/ruby/MassMatrix3.i#L99-L105